docs/patterns/tools/use-tools/use-specific-actions.mdx

```diff
@@ -10,112 +10,124 @@ description: "Each Tool (like GitHub, Slack, etc.) comes with many Actions. You
 </Tip>
 
 ### Specifying Actions
+
 `GITHUB_CREATE_AN_ISSUE` and `GITHUB_COMMIT_EVENT` are action IDs for actions in the GitHub Tool
+
 <CodeGroup>
-```python Python
-from composio import ComposioToolSet, App, action
+  ```python Python
+  from composio_langchain import ComposioToolSet, Action
 
-toolset = ComposioToolSet()
+  toolset = ComposioToolSet()
 
-# can pass multiple actions
-tools = toolset.get_tools(actions=[Action.GITHUB_CREATE_AN_ISSUE,Action.GITHUB_COMMIT_EVENT])
-```
+  # can pass multiple actions
```

Comment:
The Python example (lines 23-25) shows filtering with a single action while the comment above mentions "can pass multiple actions". Consider updating either the comment or the example to be consistent.